ClientLib/src/main/java/com/o3dr/android/client/Drone.java

```diff
@@ -481,7 +481,7 @@ public void generateDronie() {
     }
 
     public void arm(boolean arm) {
-        DroneStateApi.arm(this, arm);
+        DroneStateApi.arm(this, arm, false);
```

Comment:
@squilter since the original method signature is still available, this change can be reverted.